specs/remove-hypervisor-dependency.md

```diff
@@ -0,0 +1,45 @@
+# Remove hypervisor dependencies
+
+## Scope
+We want to make Open vStorage a real storage product which no longer depends on a hypervisor. In previous versions we always assumed that the Volume Driver ran together with a hypervisor (VMware/KVM). With the Edge this no longer the case as the edge client runs on the compute nodes (hypervisor) and the volume driver runs separately on a different node.
+With th FR we want to break this hypervisor dependency. This means you can now install Open vStorage and create a vdisk and add the hypervisor later.
+
+## OVS setup
+* Remove the question(s) and code about the hypervisor (VMware and ESXi)
+
+## Hypervisors
+### NFS
+* Remove the NFS path for VMware (VMware will be supported through iSCSI)
+### KVM
+* For KVM we need to still support the FUSE filesystem approach
+* When creating a vDisk on the fuse layer we don't interact with th hypervisor to find out which VM it is attached too.
+* .raw only
+### Edge
+* The Edge should be able to work without the FUSe layer
```

Comment:
Wondering why this requirement is here as I think we don't want to do any changes to the Edge today? If I am not mistaken the Edge control path uses same functions as filesystem (like eg fs.create_volume).